src/app1.py

```
@@ -4,34 +4,40 @@
 
-from logger_config import configure_logging
+sys.path.insert(0, str(Path(__file__).parent))
+pymysql.install_as_MySQLdb()
+
+# Load environment variables before any other imports
 
 _env_file_path = str(Path(__file__).parent.parent.parent / ".env")
-load_dotenv(_env_file_path)
+try:
+    load_dotenv(_env_file_path)
+except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)